tapalcatl_server/handler.go

```diff
@@ -31,10 +30,10 @@ type BufferManager interface {
 // a log parser will pick this up and use it to persist metrics
 // the string functions here are specific to the format used and should be updated with care
 
-type reqResponseState int32
+type ReqResponseState int32
```

Comment:
I initially tried to marshall a struct, which required some capitalization of names. And so I didn't have to think about it, I just capitalized everything related here.
I since switched approaches to using an explicit map object in order to accumulate state along the way in a flexible way, so this is no longer necessary, but I left these changes in.